demos/notebooks/demo_pipeline_cnmfE.ipynb

```diff
@@ -288,17 +443,44 @@
    "source": [
-    "# compute some summary images (correlation and peak to noise)\n",
-    "cn_filter, pnr = cm.summary_images.correlation_pnr(images[::1], gSig=gSig[0], swap_dim=False) # change swap dim if output looks weird, it is a problem with tiffile\n",
-    "# inspect the summary images and set the parameters\n",
-    "nb_inspect_correlation_pnr(cn_filter, pnr)"
+    "def key_params_cnmfe(cnmfe_model):\n",
+    "    \"\"\"\n",
+    "    Convenience function to return critical parameters given CNMFE estimator object.\n",
+    "    Returns dict with values of rf, stride, gSig, gSiz, merge_threshold, min_corr, min_pnr\n",
+    "    \n",
+    "    Note: \n",
+    "    gSiz is included because it depends on gSig and you want to make sure to change it when you change gSig.\n",
+    "    These are not set in stone: tweak for your own needs!\n",
+    "    \"\"\"\n",
+    "    rf = cnmfe_model.params.patch['rf']\n",
+    "    stride = cnmfe_model.params.patch['stride']\n",
+    "    gSig = cnmfe_model.params.init['gSig']\n",
+    "    gSiz = cnmfe_model.params.init['gSiz']\n",
+    "    merge_thr = cnmfe_model.params.merging['merge_thr']\n",
+    "    min_corr = cnmfe_model.params.init['min_corr']\n",
+    "    min_pnr = cnmfe_model.params.init['min_pnr']\n",
+    "    \n",
+    "    \n",
+    "    key_params = {'min_corr': min_corr,    \n",
+    "                  'min_pnr': min_pnr,\n",
+    "                  'rf': rf, \n",
+    "                  'stride': stride,\n",
+    "                  'gSig': gSig,\n",
+    "                  'gSiz': gSiz,\n",
+    "                  'merge_thr': merge_thr}\n",
+    "    \n",
+    "    return key_params"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "You can inspect the correlation and PNR images to select the threshold values for `min_corr` and `min_pnr`. The algorithm will look for components only in places where these value are above the specified thresholds. You can adjust the dynamic range in the plots shown above by choosing the selection tool (third button from the left) and selecting the desired region in the histogram plots on the right of each panel."
+    "## Inspect summary images and set parameters\n",
+    "### Correlation-pnr plot\n",
+    "For CNMFE, Caiman uses the correlation and peak-to-noise (PNR) ratio for initialization, which will both tend to be high in regions that contain neurons. Hence, we set a threshold for both quantitites to remove the low correlation/low pnr regions, and highlight the regions higher in both metrics, those regions most likely to contain neuronal activity. \n",
+    "\n",
+    "First, we calculate the correlation and pnr maps of the raw motion corrected movie after filtering with a mean-centered Gaussian with standard deviation `gSig` (for more information, see the sidebar below). These calculation can be computationally and memory demanding for large datasets, so we subsample if there are many thousands of frames:"
```

Comment:
These calculation -> These calculations
"can be computationally and memory demanding" -- I am not sure if this is grammatical